health/health.d/vernemq.conf

```diff
@@ -83,7 +88,11 @@ component: VerneMQ
     every: 1m
      warn: $this > 0
     delay: up 5m down 5m multiplier 1.5 max 1h
-     info: amount of traffic dropped during communication with the cluster nodes in the last minute
+     info: Amount of traffic dropped during communication with the cluster nodes in the last minute. \
+           The outgoing cluster buffer is full. \
+           Verne is experiencing problems with inter-node message delivery. \
```

Comment:
The same suggestion given above, I repeat here, please, change `Verne` to `VerneMQ`.